### DISCUSSION TITLE
RHEL8 Support?

### DISCUSSION THREAD

**Discussion by imskyyc** [oven-sh/bun#1271] [Q&A]
I'm looking to run a few applications under the Bun runtime on RHEL8, problem is the Bun CLI or bun itself requires `glibc=2.29`, which RHEL only provides support for up to glibc=2.28. Any ideas how to even initialize a project or run bun on RHEL 8?

**Comment by einar-hjortdal** [comment]
This has been discussed ad nauseam.

RHEL 8 is on life support, the focus should instead o be on RHEL 9.

RHEL 9 comes with the correct version of glibc but disabled uring (Red Hat disabled it for security reasons). So even if glibc was staticly linked, lib uring would be blocking.

Best option at the moment is running it in a ubuntu virtual machine, on either RHEL versions.
